scripts/coreos-installer-service

```diff
@@ -10,6 +10,9 @@ PERSIST_KERNEL_NET_PARAMS=("ipv6.disable" "net.ifnames" "net.naming-scheme")
 # List from https://www.mankier.com/7/dracut.cmdline#Description-Network
 PERSIST_DRACUT_NET_PARAMS=("ip" "ifname" "rd.route" "bootdev" "BOOTIF" "rd.bootif" "nameserver" "rd.peerdns" "biosdevname" "vlan" "bond" "team" "bridge" "rd.net.timeout.carrier" "coreos.no_persist_ip" "coreos.force_persist_ip")
 
+# Console params to persist
+PERSIST_CONSOLE_PARAMS=("console")
```

Comment:
@jlebon in https://github.com/coreos/coreos-installer/pull/605#discussion_r890512361:
> I have mixed feelings about this. On the one hand, I think as a heuristic this makes sense (I know I suggested it myself at least once), but OTOH this is adding more kargs magic which increases the divide between the `coreos.inst` path vs manual installs.
> 
> I guess one thing that helps is that in the `coreos.inst` path, we automatically reboot which strongly implies that the console setting should be the same.
> 
> Maybe what would help too is if we had a `coreos-installer install` option like `--copy-console-kargs`. Then we could frame it as "the `coreos.inst` path implies `--copy-console-kargs`, which can also be enabled in a manual install". (On that note, maybe we could even also have `--copy-network-kargs` and `--copy-s390x-kargs` too and drain everything here into coreos-installer proper.)
> 
> Anyway, not a blocker, just voicing some thoughts.